### ISSUE TITLE
Matching item delimiter and decimal point  (comma) messes up CSV export

### ISSUE THREAD

**Issue by simonberger** [highcharts/highcharts#7986]
CSV Export creates a bad file if the defined or calculated decimal point matches the defined item delimiter.
In a change or patch some time ago you already set the default delimiter to ";" if the decimal point is "," but the it still fails if the delimiter has been defined by the user.

Codeblock
https://github.com/highcharts/highcharts/blob/master/js/modules/export-data.src.js#L496

Unimportant optimization but I noted there can be a if / else if on this type checking too.

**Comment by pawelfus** [comment]
Hi @simonberger 

Thank you for reporting the issues. You most probably refer to #7448 issue. Could you explain the case a bit more? You mean, you set manually [`exporting.csv.itemDelimiter`](https://api.highcharts.com/highcharts/exporting.csv.itemDelimiter) to `,` and now it can break exported CSV's?

**Comment by simonberger** [comment]
@pawelfus 
Yes that looks like the issue to this fix I saw.
But as described it fixes just the defaukt behaviour. If I force the usage of a comma it is still broken.
I modified the example of your linked issue to demonstrate the problem.

http://jsfiddle.net/3hgd5hu2/1/

Quotes has to be added to those columns. For example like this https://github.com/simonberger/export-csv/commit/8a21717229566a2f42846a3424b9ed527d76d871
